srcpkgs/python3-pandas-stubs/template

```diff
@@ -0,0 +1,17 @@
+# Template file for 'python3-pandas-stubs'
+pkgname=python3-pandas-stubs
+version=1.5.2.221213
+revision=1
+build_style=python3-pep517
+hostmakedepends="python3-poetry-core"
+depends="python3-typing_extensions python3-pandas"
```

Comment:
```suggestion
depends="python3-typing_extensions python3-pandas"
checkdepends="python3-pytest"
```
and use the github tarball because it has tests

Reply:
To implement this at a minimum will require new packages for:
python3-xarray
python3-pyarrow

and an update to:
python3-SQLAlchemy

Reply:
You can just disable tests in that case and mention why in a comment next to it